ui/pages/confirmations/confirmation/templates/add-ethereum-chain.js

```diff
@@ -358,43 +383,84 @@ function getValues(pendingApproval, t, actions, history, data) {
           },
         },
       },
-      {
-        element: 'TruncatedDefinitionList',
-        key: 'network-details',
-        props: {
-          title: t('networkDetails'),
-          tooltips: {
-            [t('networkName')]: t('networkNameDefinition'),
-            [t('networkURL')]: t('networkURLDefinition'),
-            [t('chainId')]: t('chainIdDefinition'),
-            [t('currencySymbol')]: t('currencySymbolDefinition'),
-            [t('blockExplorerUrl')]: t('blockExplorerUrlDefinition'),
-          },
-          warnings: {
-            [t('currencySymbol')]: data.currencySymbolWarning,
-          },
-          dictionary: {
-            [t('networkName')]: pendingApproval.requestData.chainName,
-            [t('networkURL')]: pendingApproval.requestData.rpcUrl
-              .toLowerCase()
-              ?.includes(`/v3/${infuraProjectId}`)
-              ? pendingApproval.requestData.rpcUrl
-                  .replace(`/v3/${infuraProjectId}`, '')
+
+      process.env.CHAIN_PERMISSIONS
```

Comment:
Also here, less duplication and easier to maintain if we only branch in the specific places where the values differ?

Reply:
Done here https://github.com/MetaMask/metamask-extension/pull/25777/commits/e87feb11f778de6e03ad95819414cbb2233c4b80